lib/ansible/plugins/doc_fragments/action_core.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
How about renaming this file to `_action_core.py` to emphasize that it is private (and shouldn't be used by collections)?

Reply:
underscores indicate deprecation, maybe a . ?

Reply:
Underscores are used for both :) (Which is somewhat unfortunate...) At least outside core, underscores are only used for private, since file-level deprecation is done exclusively by routing.